public/index.php

```diff
@@ -4,6 +4,26 @@
  * @copyright Copyright (c) 2014 Zend Technologies USA Inc. (http://www.zend.com)
  */
 
+if (ini_get('display_errors')) {
+    /*
+     * PHP by default returns HTTP 200 even when exceptions are thrown.
+     *
+     * Now by default return HTTP 500 so any API client can trust in the HTTP response code.
+     */
+    http_response_code(500);
```

Comment:
In my tests this was enough for capture normal exceptions while the exception handler was for capture php errors

I don't know if this setting has effects on apigility admin